src/app/modules/item/components/item-header/access-code-view/access-code-view.component.ts

```diff
@@ -13,6 +13,7 @@ import { ItemData } from '../../../services/item-datasource.service';
 export class AccessCodeViewComponent {
 
   @Input() itemData?: ItemData;
+  @Output() reload = new EventEmitter<void>();
```

Comment:
A output called  "reload" for an AccessCodeViewComponent is not really clear. (which may be used in different context)

Reply:
fixed